ibis/common/geospatial.py

```diff
@@ -2,7 +2,8 @@
 
 import ibis.expr.datatypes as dt
```

Comment:
`absolufy` doesn't seem to handle plain imports, but I guess that's OK.

Reply:
Hey, thanks for using `absolufy-imports` - I don't think plain imports can be written as relative, can they?

Reply:
Hi @MarcoGorelli! Thanks for chiming in!

It should be possible to rewrite them an relative, in this case `from ..expr import datatypes as dt`

Reply:
true, but that would change it from a direct import to a from import, not sure if people would want that...worth considering anyway, thanks!